src/plugins/importFromJSONAction/ImportFromJSONAction.js

```diff
@@ -71,8 +72,10 @@ export default class ImportAsJSONAction {
 
   onSave(object, changes) {
     const selectFile = changes.selectFile;
-    const objectTree = selectFile.body;
-    this._importObjectTree(object, JSON.parse(objectTree));
+    const jsonTree = selectFile.body;
+    const objectTree = JSON.parse(jsonTree, filter__proto__);
```

Comment:
Can we use Vue's toRawutility here?
Something like:
return JSON.parse(toRaw(jsonTree)